### PR TITLE
Change absolute units(px) to relative units(rem)

### PR DESCRIPTION
**Pull request by Chandu-4444**
Hey, I've changed the units mentioned here #7548 . I'm a beginner, please let me know if anything's wrong.

### CODE REVIEW

**Review by ivanetra — APPROVED** [review]
These changes can be merged!

**Comment by jonkoops** [comment]
I believe this might actually be a breaking change as the `rem` unit is relative to the font-size of the root element of the page. Since this can be different for each page Leaflet is embedded in this might cause unexpected side-effects.

**Comment by jonkoops** [comment]
I don't believe compatibility with Internet Explorer needs to be a problem as we can provide a [fallback value](https://w3bits.com/rem-fallback/).

**Comment by Malvoz** [comment]
Agree with @jonkoops, please consider reopening this.

File: dist/leaflet.css [hunk 1]
```diff
@@ -431,7 +431,7 @@ svg.leaflet-image-layer.leaflet-interactive path {
 	border-top: none;
 	line-height: 1.1;
 	padding: 2px 5px 1px;
-	font-size: 11px;
+	font-size: 0.69rem;
```
Comment (Malvoz) [inline dist/leaflet.css:434]: @Chandu-4444 would you like to change this to also include the `px` unit declarations as a fallback per https://github.com/Leaflet/Leaflet/pull/7571#issuecomment-961001947?

Comment (Chandu-4444) [reply dist/leaflet.css:434]: Yeah sure!

**Comment by mourner** [comment]
I tried reopening but it doesn't allow me to, saying that the repo of the PR has been deleted — want to update @Chandu-4444, or submit a new PR if it's not easily possible?

**Comment by Chandu-4444** [comment]
> I tried reopening but it doesn't allow me to, saying that the repo of the PR has been deleted — want to update @Chandu-4444, or submit a new PR if it's not easily possible?

I'll submit a new PR then.

**Pull request closed without merging**